$NetBSD: patch-CMakeLists.txt,v 1.1 2018/08/12 14:25:09 nia Exp $

Avoid checking for a C++ compiler, since C++ isn't required for a
libretro build.

vfs-dirent peeks into non-standard structures and also isn't required
for a libretro build.

--- CMakeLists.txt.orig	2018-04-14 20:49:19.000000000 +0000
+++ CMakeLists.txt
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.8.11)
 if(POLICY CMP0025)
 	cmake_policy(SET CMP0025 NEW)
 endif()
-project(mGBA)
+project(mGBA C)
 set(BINARY_NAME mgba CACHE INTERNAL "Name of output binaries")
 if(NOT MSVC)
 	set(GCC_STD "c99")
@@ -231,7 +231,7 @@ elseif(UNIX)
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
 	endif()
 
-	list(APPEND CORE_VFS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/util/vfs/vfs-fd.c ${CMAKE_CURRENT_SOURCE_DIR}/src/util/vfs/vfs-dirent.c)
+	list(APPEND CORE_VFS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/util/vfs/vfs-fd.c)
 	file(GLOB OS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/platform/posix/*.c)
 	source_group("POSIX-specific code" FILES ${OS_SRC})
 endif()
